### PR TITLE
created InterestsContext added code to Context

### LINKED ISSUE #34: 5 1/2. Make InterestsContext.java and add various code
Finish #22 first
Add code here: https://github.com/lysanderp41/CodeUSummer2017/blob/f95619a95c20252b3ff1b8d5659845d49d186cf7/src/codeu/chat/client/core/Context.java#L22
Make InterestsContext.java like [UserContext.java](https://github.com/lysanderp41/CodeUSummer2017/blob/ff639447ffe6d2e2f46e8781a0ba7be10fbc1c18/src/codeu/chat/client/core/UserContext.java)

### PR DESCRIPTION
**Pull request by mlk209**
Im not really sure what goes in Context? the first method i think is adding a new user and the second i think displays all the users? So I don't really think we are creating a new object of interests because each user should only have one object of interests that it maps to. Let me know if im understanding the code wrong! thanks! Also, for interests context I was kinda confused on the conversation context... 

closes #34

### CODE REVIEW

**Comment by nvaa36** [comment]
Honestly not sure either, but I think this looks good as long as it builds fine.

**Review by nvaa36 — APPROVED** [review]

**Comment by mlk209** [comment]
There was an error in the context.java file 
src/codeu/chat/client/core/Context.java:54: error: cannot find symbol
    return view.getInterests();
               ^
  symbol:   method getInterests()
  location: variable view of type View
1 error
1 warning
Build FAILED

but i think thats because i hadnt added the command in the view file yet?

**Comment by mlk209** [comment]
Sorry, now its giving me a different error 
src/codeu/chat/client/core/Context.java:54: error: incompatible types: Collection<Interests> cannot be converted to Interests
    return view.getInterests();

**Comment by nvaa36** [comment]
I would just delete that function. I'm honestly not sure we'll need and we can add it later if we do.

File: src/codeu/chat/client/core/Context.java [hunk 1]
```diff
@@ -49,7 +50,9 @@ public UserContext create(String name) {
     }
     return users;
   }
-
+  public Interests getInterests (){
```
Comment (nvaa36) [inline src/codeu/chat/client/core/Context.java:53]: Not sure you need this function.

**Pull request closed without merging**